libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/internal/NavigationAppUserTurnstileEvent.kt

```diff
@@ -1,12 +1,10 @@
 package com.mapbox.navigation.metrics.internal
 
-import androidx.annotation.Keep
 import com.google.gson.Gson
 import com.mapbox.android.telemetry.AppUserTurnstile
 import com.mapbox.navigation.base.metrics.MetricEvent
 import com.mapbox.navigation.base.metrics.NavigationMetrics
 
-@Keep
```

Comment:
We removed this `@Keep` but don't see it in `libnavigation-metrics` `proguard-rules.pro`, we should either leave the `@Keep` or add the Proguard rule.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/2643/files#diff-e48086f49707cba8519b9bea42f2e1e2R24

Reply:
`-keep class com.mapbox.navigation.core.telemetry.events.** {*;}` that’s only `events` package, this is a different one that lives in a different module (`libnavigation-metrics`) 👀 `package com.mapbox.navigation.metrics.internal`